src/components/ConnectWeb3/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import styled from '@emotion/styled';
+import Modal from '../../future-hopr-lib-components/Modal';
+import WalletButton from '../../future-hopr-lib-components/Button/wallet-button';
+
+// Store
+import { useAppDispatch } from '../../store';
+import { web3Actions } from '../../store/slices/web3';
+
+// wagmi
+import { useAccount, useConnect, useDisconnect } from 'wagmi';
+import { InjectedConnector } from 'wagmi/connectors/injected';
+
+const ConnectWalletContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  margin-top: 8px;
+  p {
+    margin-top: 48px;
+  }
+`;
+
+export default function ConnectWeb3() {
+  const dispatch = useAppDispatch();
+  const [chooseWalletModal, set_chooseWalletModal] = useState(false);
```

Comment:
`set_chooseWalletModal` are we going to use this convention? I think this can look a bit out of place in our repo but if we do this then all are sets have to be like this.

Reply:
Yes, this way it is a lot easier to find what you are looking for with case sensitive search, and also you can simply copy pase the const name with addition of _ instead of changing the 1st letter to caps